rotkehlchen/db/filtering.py

```diff
@@ -169,6 +169,30 @@ def prepare(self) -> tuple[list[str], list[Any]]:
         return [query], bindings
 
 
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
+class DBTransactionsPendingDecodingFilter(DBFilter):
+    """
+    This filter is used to find the ethereum transactions that have not been decoded yet
+    using the query in `TRANSACTIONS_MISSING_DECODING_QUERY`. It allows filtering by addresses
+    and chain.
+    """
+    addresses: Optional[list[ChecksumEvmAddress]]
+    chain_id: Optional[ChainID]
+
+    def prepare(self) -> tuple[list[str], list[Any]]:
+        query_filters = ['B.tx_hash is NULL']
+        bindings: list[Union[int, ChecksumEvmAddress]] = []
+        if self.addresses is not None:
+            bindings += self.addresses + self.addresses
```

Comment:
No need to append to an emty list. Easier to understand:

```suggestion
            bindings = [*self.addresses, *self.addresses]
```